### DISCUSSION TITLE
What are the complete syntax rules for emphasis?

### DISCUSSION THREAD

**Discussion by italicize** [community/community#22854] [New to GitHub]
<p>Can someone point me to <em>all</em> the rules about bold and italic text in GitHub Flavored Markdown? I already found two surprises:</p>
<ul>
<li>An underscore after a letter doesn’t begin italic text. Try _italic_ but_not italic_</li>
<li>An asterisk after a letter and before a parenthesis doesn’t begin italic text. Try regular*italic* but*(why not italic?)*</li>
</ul>

**Comment by ThirdEyeRose** [comment]
<p>Hi <a class="mention" href="/italicize">@italicize</a>,</p>
<p>GitHub uses CommonMark as the Markdown processor, so you can view all of the <a href="https://spec.commonmark.org/0.28/#emphasis-and-strong-emphasis" rel="nofollow noopener">rules for emphasis in the CommonMark specifications</a>.</p>
<p>Does this help?</p>
<p>Thanks!</p>

**Comment by italicize** [comment, marked as answer]
<p>The <a href="https://github.github.com/gfm/#emphasis-and-strong-emphasis" rel="nofollow noopener">complete specification</a> has 17 rules about when emphasis begins and ends. To summarize, I wrote <a href="https://gist.github.com/italicize/92f0172c1dd097e94a6b4ed79c5c5e49" rel="nofollow noopener">three rules</a> for myself:</p>
<ol>
<li>Use either asterisks or underscores to emphasize a character next to a space, punctuation, or the beginning or end of a line. Put an emphasis mark immediately next to a character, with no space between.</li>
</ol>
<ul>
<li>
<em>Exception.</em> Special punctuation marks override this rule, including a backslash, backtick, and square brackets which cause an asterisk or underscore to be rendered as a character and ignored as an emphasis mark.</li>
</ul>
<ol start="2">
<li>Use an asterisk to emphasize letters and numbers next to letters and numbers. By contrast, an underscore is not an emphasis mark between letters and numbers, as in not_italic.</li>
<li>To emphasize punctuation next to letters and numbers, use HTML tags. By contrast, an asterisk and underscore cannot emphasize punctuation next to an unemphasized letter or number.</li>
</ol>
<p>To see these rules with examples, see <a href="https://gist.github.com/italicize/92f0172c1dd097e94a6b4ed79c5c5e49" rel="nofollow noopener">Emphasis in GitHub Flavored Markdown</a>.</p>
